crates/ruff_python_formatter/src/expression/expr_lambda.rs

```diff
@@ -52,6 +52,11 @@ impl FormatNodeRule<ExprLambda> for FormatExprLambda {
             ]
         )
     }
+
+    fn fmt_dangling_comments(&self, _node: &ExprLambda, _f: &mut PyFormatter) -> FormatResult<()> {
+        // Override. Danlging comments are handled in `fmt_fields`.
```

Comment:
```suggestion
        // Override. Dangling comments are handled in `fmt_fields`.
```
Can I get this even with the auto-merge?